cardano-cli/src/Cardano/CLI/EraBased/Run/Key.hs

```diff
@@ -46,49 +48,55 @@ import           Control.Monad.Trans.Except.Extra (firstExceptT, hoistEither, le
 import           Data.Bifunctor (Bifunctor (..))
 import           Data.ByteString (ByteString)
 import qualified Data.ByteString as BS
+import           Data.Function
 import           Data.Text (Text)
 import qualified Data.Text as Text
 import qualified Data.Text.Encoding as Text
 import           System.Exit (exitFailure)
 
 runKeyCmds :: ()
-  => KeyCmds era
+  => Cmd.KeyCmds era
   -> ExceptT KeyCmdError IO ()
 runKeyCmds = \case
-  KeyGetVerificationKey skf vkf ->
-    runGetVerificationKeyCmd skf vkf
-  KeyNonExtendedKey evkf vkf ->
-    runConvertToNonExtendedKeyCmd evkf vkf
-  KeyConvertByronKey mPassword keytype skfOld skfNew ->
-    runConvertByronKeyCmd mPassword keytype skfOld skfNew
-  KeyConvertByronGenesisVKey oldVk newVkf ->
-    runConvertByronGenesisVerificationKeyCmd oldVk newVkf
-  KeyConvertITNStakeKey itnKeyFile outFile ->
-    runConvertITNStakeKeyCmd itnKeyFile outFile
-  KeyConvertITNExtendedToStakeKey itnPrivKeyFile outFile ->
-    runConvertITNExtendedToStakeKeyCmd itnPrivKeyFile outFile
-  KeyConvertITNBip32ToStakeKey itnPrivKeyFile outFile ->
-    runConvertITNBip32ToStakeKeyCmd itnPrivKeyFile outFile
-  KeyConvertCardanoAddressSigningKey keyType skfOld skfNew ->
-    runConvertCardanoAddressSigningKeyCmd keyType skfOld skfNew
-
-runGetVerificationKeyCmd :: ()
-  => SigningKeyFile In
-  -> VerificationKeyFile Out
+  Cmd.KeyVerificationKeyCmd cmd ->
+    runVerificationKeyCmd cmd
+  Cmd.KeyNonExtendedKeyCmd cmd ->
+    runNonExtendedKeyCmd cmd
+  Cmd.KeyConvertByronKeyCmd cmd ->
+    runConvertByronKeyCmd cmd
+  Cmd.KeyConvertByronGenesisVKeyCmd cmd ->
+    runConvertByronGenesisVKeyCmd cmd
+  Cmd.KeyConvertITNKeyCmd cmd ->
```

Comment:
Can you find out if we need the ITN functionality anymore? If not we can remove all of that code.